aikau/src/main/resources/alfresco/forms/controls/NumberSpinner.js

```diff
@@ -161,30 +224,59 @@ define(["alfresco/forms/controls/BaseFormControl",
        */
       isNumberValidator: function alfresco_forms_controls_FormControlValidationMixin__isNumberValidator(validationConfig) {
          var isValid = false;
-         try {
-            var value = this._removeCommasAndSpaces(this.wrappedWidget.textbox.value),
-               isNumber = this._valueIsNumber(value);
-            isValid = isNumber || (this.permitEmpty && value === "");
-         } catch (e) {
+         try 
+         {
+            var textBoxValue = this.wrappedWidget.textbox.value;
```

Comment:
No. Nothing should access `textbox.value` outside of the widget. `this.wrappedWidget.get("value")` will give you a number or null, and so should be used to check for a valid number.